README.md

```diff
@@ -161,6 +161,14 @@ shown below. Can't see yours? Have a look at the [extra platform instructions](h
    ```
    To update the Starship itself, rerun the above script. It will replace the current version without touching Starship's configuration.
 
+
+   **Note** - The defaults of the install script can be overridden see the build in help.
+  **Note** - The defaults of the install script can be overridden see the built-in help.
```

Comment:
The line is duplicated and the old indentation was correct:
```suggestion
   **Note** - The defaults of the install script can be overridden see the built-in help.
```

Reply:
😅 look like I mistook `suggestion` block as `diff`